frontend/src/style/globals.css

```diff
@@ -3,6 +3,8 @@
   --brown: #9b4c10;
   --pink: #d6606d;
   --blue: #7796b2;
+  --black: #333333;
+  --white: #FFFAF0; 
```

Comment:
I ran lighthouse on this and if we keep --white at #FFFFFF we're still 92% accessibility so that should be fine. 

Change that value just because it looks a bit nicer and still won't drop our readability and then this issue is done :)